### ISSUE TITLE
Windows Resizing

### ISSUE THREAD

**Issue by ATCUSA** [mbridak/WinterFieldDayLogger#21]
The window is not resizing and starts out quite compact and cutting off text. This includes the main window as well as the settings window.

The window should be re-sizable to allow seeing all text on different displays.

**Screenshots**
![image](https://user-images.githubusercontent.com/3581228/213559964-254d8cee-3098-4bee-b073-300f1c93cb6c.png)

![image](https://user-images.githubusercontent.com/3581228/213560046-e63f3f28-4721-40e3-aa9a-e5602e5066fc.png)

![image](https://user-images.githubusercontent.com/3581228/213560098-c76add5f-d096-4d00-86bc-8ac63cb298cb.png)

![image](https://user-images.githubusercontent.com/3581228/213560149-adb5fdf0-035c-4fc0-bd2c-8f9d4d242182.png)

**Desktop (please complete the following information):**
 - OS: Linux Kubuntu
 - Version: 22.04.1

**Comment by mbridak** [comment]
Hi, looks like it's not finding the font that should have been packaged with it, so it's choosing something close that's on the computer.

I'll try and track it down. Thanks for the report.

If for some reason I fall Ill or for some reason not able to fix this before WFD, you should be able to resolve the issue locally by copying the font found [here](https://github.com/mbridak/WinterFieldDayLogger/blob/main/wfdlogger/data/JetBrainsMono-Regular.ttf) to ~/.fonts/

**Comment by mbridak** [comment]
Having trouble replicating the font problem.
I'll try installing kubuntu in a VM and try again.

BTW Did you install it with pip? Or did you run it from source?

**Comment by ATCUSA** [comment]
I used pip to install. I can try running from source as well. Thanks for looking into this. ---- On Thu, 19 Jan 2023 19:38:52 -0800  ***@***.***  wrote ----
Having trouble replicating the font problem.
I'll try installing kubuntu in a VM and try again.
BTW Did you install it with pip? Or did you run it from source?

**Comment by mbridak** [comment]
I installed kubuntu 22.04.1 LTS into a VM, installed pending updates and installed pip.
Used pip to install wfdlogger.

It's detecting and using the packaged font. I'm not sure why it's doing that for you.

It's true it won't resize. That ain't changing anytime soon. So far I've been too lazy to figure that out.
Maybe some day in the future, I'll get a 4k monitor and be properly motivated.
 
![Screenshot_20230119_215354](https://user-images.githubusercontent.com/49108421/213628211-0bfc4492-23bb-4901-9840-44fc742352ed.png)
![Screenshot_20230119_215646](https://user-images.githubusercontent.com/49108421/213628224-4172fdb0-a5d7-472d-b41a-0fd15d15c12b.png)

**Comment by mbridak** [comment]
I think I found and can replicate the nature of the problem.
If I go into my desktops accessibility settings and turn on large text I get this.

![Screenshot from 2023-01-20 09-54-56](https://user-images.githubusercontent.com/49108421/213772552-3627427a-9bc6-4e60-b79f-6da76e1d7430.png)

Which is fairly close to your display.

Getting the dialogs to dynamically resize is fairly straight forward. I've finished most of them already.
But the main screen is another story. There's no way I could get it done by the event time.
